Tests/FluentAssertions.Specs/Primitives/DateTimeAssertionSpecs.cs

```diff
@@ -2245,6 +2245,24 @@ public void Should_support_chaining_constraints_with_and()
             // Assert
             action.Should().NotThrow();
         }
+
+        [Fact]
+        public void Should_throw_because_of_assertion_failure()
```

Comment:
@dennisdoomen  I'd suggest it's possible to check whether exception message doesn't contain any markers of failed formatting (`**WARNING**` or whatever) or check that exception message contains formatted message. What would you prefer?
In any case, the point of test is to assert that error is thrown because of assertion failure, not formatting error.